spec/controllers/two_factor_authentication/piv_cac_verification_controller_spec.rb

```diff
@@ -194,6 +210,13 @@
         expect(@analytics).to receive(:track_mfa_submit_event).
           with(submit_attributes)
 
+        expect(@irs_attempts_api_tracker).to receive(:track_event).with(
+          :mfa_login_piv_cac,
+          success: false,
+          subject_dn: 'bad-dn',
+          failure_reason: { type: 'user.piv_cac_mismatch' },
+        )
+
         expect(@analytics).to receive(:track_event).
```

Comment:
Is it necessary to have this expect block in both places (line 131 and line 213)? One is inside the `before` statement. Is there a significance to that?

Reply:
Not necessary you are right reverting that.